crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/service/CrimeEvidenceDataService.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.laa.crime.crowncourt.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.justice.laa.crime.crowncourt.client.CrimeEvidenceClient;
+import uk.gov.justice.laa.crime.crowncourt.common.Constants;
+import uk.gov.justice.laa.crime.crowncourt.config.ServicesConfiguration;
+import uk.gov.justice.laa.crime.crowncourt.model.ApiCalculateEvidenceFeeRequest;
+import uk.gov.justice.laa.crime.crowncourt.model.ApiCalculateEvidenceFeeResponse;
+
+import java.util.Map;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class CrimeEvidenceDataService {
+
+    public static final String RESPONSE_STRING = "Response from Court Data API: %s";
```

Comment:
Since `RESPONSE_STRING` is only used in this class it should probably be `private`, rather than a global constant